pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -32,6 +31,7 @@ import (
 	"istio.io/istio/pilot/pkg/networking/plugin"
 	"istio.io/istio/pilot/pkg/networking/util"
 	"istio.io/istio/pilot/pkg/proxy/envoy/v1"
+	"istio.io/istio/pkg/log"
```

Comment:
not sure why this is added, looks unused?

Reply:
it is added in place of `github.com/prometheus/common/log` It was used on line 146.